frontend/src/services/AuthService.ts

```diff
@@ -128,6 +128,13 @@ function storeFamUser(famLoginUser: FamLoginUser | null | undefined) {
     }
 }
 
+const userHasAccessByRole = (role: string): boolean => {
```

Comment:
Very minor, can we name it as "hasAccessRole"?

Reply:
Thanks, I'll change it.